docs/src/lib/conversion.md

```diff
@@ -21,6 +21,8 @@ convert(::Type{HPOLYGON}, ::VPolygon) where {HPOLYGON<:AbstractHPolygon}
 convert(::Type{Hyperrectangle}, ::Interval)
 convert(::Type{Interval}, ::AbstractHyperrectangle)
 convert(::Type{Interval}, ::LazySet{N}) where {N<:Real}
+convert(::Type{Hyperrectangle}, cpa::CartesianProductArray{N, <:AbstractHyperrectangle{N}}) where {N<:Real}
```

Comment:
Unfortunately, `Documenter` does not understand nested `<:`. Maybe try to explicity add a type parameter in the function, as in:
```julia
convert(::Type{Hyperrectangle}, cpa::CartesianProductArray{N, H}) where {N<:Real, H<:AbstractHyperrectangle{N}}
```